fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -671,6 +675,13 @@ protected void addResourceHttpHeaders(final FedoraResource resource,
             servletResponse.addHeader("Accept-Ranges", "bytes");
             servletResponse.addHeader(CONTENT_DISPOSITION, dispositionBuilder.build().toString());
         }
+        // Add ocflPath header if desired
+        if (ocflPropsConfig.isShowPath()) {
+            final var contentPath = resource.getStorageRelativePath();
+            if (contentPath != null) {
+                servletResponse.addHeader("Fedora-Ocfl-Path", contentPath.toString());
```

Comment:
We're pretty inconsistent about it in this file, but it seems like we should have a constant for the header name.